app/test_util.py

```diff
@@ -69,13 +69,13 @@ def wait_for_css_to_be_clickable(self, css, timeout=5):
             expected_conditions.element_to_be_clickable((By.CSS, css))
         )
 
-    def scroll_to_element_and_click(self, element):
-        self.execute_script("arguments[0].scrollIntoView();", element)
-        return element.click()
+    def scroll_to_element_and_click(self, element, timeout=5):
+        self.execute_script("arguments[0].scrollIntoView(true);", element)
+        ActionChains(self).move_to_element(element).click().perform()
 
     def click_xpath(self, xpath):
         element = self.wait_for_xpath_to_be_clickable(xpath)
-        return self.scroll_to_element_and_click(element)
+        self.scroll_to_element_and_click(element)
```

Comment:
Shouldn't this return whatever click returns?

Reply:
We don't use it, but probably good practice.